src/membrane-keypad-element.ts

```diff
@@ -182,4 +203,81 @@ export class MembraneKeypadElement extends LitElement {
       );
     }
   }
+
+  private defineKey(key: number) {
+    let keyName;
+    const numKey = key - 48;
+
+    switch (key) {
+      case SHIFT_KEY:
+        if (!this.shiftPressed) {
+          this.shiftPressed = true;
+        }
+        break;
+      case 48: // falls through
+      case 49: // falls through
+      case 50: // falls through
+      case 51:
+        if (this.shiftPressed) {
+          keyName = '#keyHash';
+          break;
+        }
+      // falls through
+      case 52: // falls through
+      case 53: // falls through
+      case 54: // falls through
+      case 55: // falls through
+      case 56:
+        if (this.shiftPressed) {
+          keyName = '#keyAsterisk';
+          break;
+        }
+      // falls through
+      case 57:
+        keyName = `#key${numKey}`;
+        break;
+      case 65:
+        keyName = '#keyA';
+        break;
+      case 66:
+        keyName = '#keyB';
+        break;
+      case 67:
+        keyName = '#keyC';
+        break;
+      case 68:
+        keyName = '#keyD';
+        break;
+      default:
+        console.log('Not a key');
+        break;
+    }
+    return keyName;
+  }
+
+  private keyStrokeDown(key: number) {
+    let selectedKey;
+    const keyName = this.defineKey(key);
+
+    if (keyName) {
+      selectedKey = this.shadowRoot?.querySelector(keyName);
+      selectedKey?.dispatchEvent(customKeydownEvent);
```

Comment:
How about just calling `down()` and `up()` directly to simplify things? We can change their signature to access the target element instead of an event object (after all, they only need the target element, not the entire event).